sources/packages/backend/libs/services/src/confirmation-of-enrollment/confirmation-of-enrollment.service.ts

```diff
@@ -627,7 +655,7 @@ export class ConfirmationOfEnrollmentService {
       );
     }
 
-    await this.updateCOEToDeclined(disbursementSchedule.id, auditUserId, {
+    await this.updateCOEToDeclined(disbursementSchedule, auditUserId, {
```

Comment:
Can we change this to be explicit about the parameters being expected?
Instead of providing the full `disbursementSchedule` we should probably provide the `disbursementScheduleId` and the `studentAssessmentId`.

Reply:
I can update that way, as potentially this method is expected to go through a bug fix to decline second enrolment automatically, when first is declined I thought it is going to receive more properties from disbursement object.